packages/flex-plugin-scripts/src/index.ts

```diff
@@ -59,6 +62,14 @@ const spawnScript = async (...argv: string[]) => {
     processArgs.push('--disallow-versioning');
   }
 
+  // Backwards Compatability 'npm run start'
+  if (getCwd().includes('plugin-') && !processArgs.includes(getPaths().app.name)) {
+    const port = await findPort(getDefaultPort(process.env.PORT));
+    processArgs.push('--name');
+    processArgs.push(getPaths().app.name);
+    _updatePluginPort(port, getPaths().app.name);
```

Comment:
Don't add the `port` here. Port is only required for the `start` script and so that script should add it. The only thing you should do here is adding the `--name`

Reply:
without updating the port the plugin doesn't load because the port will be incorrect/taken from the plugins.json which will be inaccurate